app/src/main/res/values/strings.xml

```diff
@@ -528,5 +528,5 @@
   <string name="bloodpressurehistory_bp_reading" translatable="false">%1$d / %2$d</string>
   <string name="bloodpressurehistory_add_bp_button">Blood Pressure</string>
   <string name="bloodpressurehistory_toolbar_title" translatable="false">%1$s, %2$s, %3$d</string>
-  <string name="bloodpressurehistory_bp_time_date">%1$s, %2$s</string>
+  <string name="bloodpressurehistory_bp_date_time" translatable="false">%1$s\n%2$s</string>
```

Comment:
It is not necessary to change the names of resource IDs to match how we're displaying a certain value. The only time we will need to change the IDs is if thing we are displaying here itself has changed (say, from a timestamp to something else).